Tests/PostgreSQLTests/PostgreSQLConnectionTests.swift

```diff
@@ -370,7 +398,8 @@ extension PostgreSQLConnection {
     static func makeTest() throws -> PostgreSQLConnection {
         let hostname: String
         #if Xcode
-        hostname = (try? Process.execute("docker-machine", "ip")) ?? "192.168.99.100"
+        //hostname = (try? Process.execute("docker-machine", "ip")) ?? "192.168.99.100"
+        hostname = "localhost"
```

Comment:
Unrelated?

Reply:
this is from my mismerge, waiting on @tanner0101  to berate me, I dont have the ability to roll back a merge or I would have already